docs/source/data-ingestion.rst

```diff
@@ -0,0 +1,68 @@
+**************
+Data Ingestion
+**************
+
+DISCO could ingest simulation metrics and analysis results into sqlite database, which
+facilitates the data sharing between researchers, and data query for further investigation.
+
+Ingest to New Database
+======================
+
+Suppose we are assigned a task which requires us to run DISCO pipeline for 
+hosting capacity analysis with snapshot simulation, and generate pipeline output directory 
+named ``/data/snapshot-output/`` with simulation and analysis result. 
+
+Run the command below, and it could help us to ingest the results into a database.
+
+.. code-block:: bash
+
+    $ disco ingest-tables --task-name "Snapshot Task" --model-inputs /data/input-models/ --database=test.sqlite /data/snapshot-output/
```

Comment:
The task name must be unique, and so I suggest using a specific example here, such as `"SFO Region P1U Snapshot"`.

I think we should suggest a convention so that all tasks look the same. `<geography> <type>`. Optional <tag>. Don't include "Task"

Reply:
Updated the docs.